test/e2e/scheduling/predicates.go

```diff
@@ -148,6 +148,78 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		verifyResult(cs, podsNeededForSaturation, 1, ns)
 	})
 
+	// This test verifies we don't allow scheduling of pods in a way that sum of local ephemeral storage limits of pods is greater than machines capacity.
+	// It assumes that cluster add-on pods stay stable and cannot be run in parallel with any other test that touches Nodes or Pods.
+	// It is so because we need to have precise control on what's running in the cluster.
+	It("validates local ephemeral storage resource limits of pods that are allowed to run [Conformance]", func() {
```

Comment:
fwiw we need to start notifying @kubernetes/sig-testing-feature-requests on [Conformance] additions.   It doesn't look like this test is version gated, so it will fail when you run the 1.8 tests on a 1.7 cluster.